src/Sulu/Bundle/MediaBundle/Api/Media.php

```diff
@@ -1108,4 +1108,21 @@ public function setFocusPointY($focusPointY)
     {
         $this->getFileVersion()->setFocusPointY($focusPointY);
     }
+
+    /**
+     * @VirtualProperty
+     * @SerializedName("previewImageId")
+     *
+     * @return ?int
+     */
+    public function getPreviewImageId()
```

Comment:
how did this work before? 🤔

Reply:
Previously it used the `previewImage` property, but since we've decided to suffix everything with `Id` if it is just an ID I have done it here as well.

Reply:
but there is no serialization configuration for a `previewImage` property? 😕

Reply:
Ah, now I remember... It is related to the other change you've mentioned in the `MediaPreviewController`. Previously a different object was returned by this controller, and that made problems because I've had to return the original object when the preview image was deleted, which caused the frontend code to be more complicated than necessary. Then I also changed the `postAction` in order to be consistent.

Reply:
okay that makes sense 🙂